tools/bin/recipebook

```diff
@@ -56,15 +56,21 @@ parser.add_argument("recipes",
                     type=str, nargs="?", default=".")
 
 parser.add_argument("-p", "--package", type=str,
+                    required=any(arg in sys.argv for arg in
```

Comment:
I paused on this because the `any` is strictly required since an non-empty list is `True` in a boolean context. However, I the `any` makes clearer what's intended.